test/addons/non-node-context/test-perf-hooks-timerify.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const common = require('../../common');
+const assert = require('assert');
+const { runInNewContext } = require(`./build/${common.buildType}/binding`);
+const { performance } = require('perf_hooks');
+
+// Check that performance.timerify() works when called from another context,
+// for a function created in another context.
+
+const check = runInNewContext(`
```

Comment:
Could you assign the string to a variable, I find this is a bit tricky to read (probably because of GitHub viewer limitations)

Reply:
![image](https://user-images.githubusercontent.com/96947/47262277-78c14780-d4b2-11e8-87cf-0a6a07dade9c.png)
It looks awkward, especially the line that starts ``` `, {```

IMHO a simple variable makes it clearer:
```js
const code = `
const { performance, assert } = data;
const timerified = performance.timerify(function() { return []; });
assert.strictEqual(timerified().constructor, Array);
'success';
`;
const check = runInNewContext(code, { performance, assert });
assert.strictEqual(check, 'success');
```